packages/dev/loaders/src/glTF/2.0/Extensions/KHR_interactivity/declarationMapper.ts

```diff
@@ -0,0 +1,1689 @@
+        //     serializedObject.signalInputs.forEach((input) => {
+        //         input.name = "in_" + input.name;
+        //     });
+        //     return serializedObjects;
+        // },
+    },
+    "flow/throttle": {
+        blocks: [FlowGraphBlockNames.Throttle],
+        outputs: {
+            flows: {
+                // out: { name: "out" },
+                err: { name: "error" },
+            },
+        },
+    },
+    "flow/setDelay": {
+        blocks: [FlowGraphBlockNames.SetDelay],
+        outputs: {
+            flows: {
+                err: { name: "error" },
+            },
+        },
+    },
+    "flow/cancelDelay": {
+        blocks: [FlowGraphBlockNames.CancelDelay],
+    },
+    "variable/get": {
+        blocks: [FlowGraphBlockNames.GetVariable],
+        validation(gltfBlock) {
+            if (!gltfBlock.configuration?.variable?.value) {
+                Logger.Error("Variable get block should have a variable configuration");
+                return false;
+            }
+            return true;
+        },
+        configuration: {
+            variable: {
+                name: "variable",
+                gltfType: "number",
+                flowGraphType: "string",
+                inOptions: true,
+                isVariable: true,
+                dataTransformer(index, parser) {
+                    return [parser.getVariableName(index[0])];
+                },
+            },
+        },
+    },
+    "variable/set": {
+        blocks: [FlowGraphBlockNames.SetVariable],
+        configuration: {
+            variable: {
+                name: "variable",
+                gltfType: "number",
+                flowGraphType: "string",
+                inOptions: true,
+                isVariable: true,
+                dataTransformer(index, parser) {
+                    return [parser.getVariableName(index[0])];
+                },
+            },
+        },
+    },
+    "variable/interpolate": {
+        blocks: [
+            FlowGraphBlockNames.ValueInterpolation,
+            FlowGraphBlockNames.Context,
+            FlowGraphBlockNames.PlayAnimation,
+            FlowGraphBlockNames.BezierCurveEasing,
+            FlowGraphBlockNames.GetVariable,
+        ],
+        configuration: {
+            variable: {
+                name: "propertyName",
+                inOptions: true,
+                isVariable: true,
+                dataTransformer(index, parser) {
+                    return [parser.getVariableName(index[0])];
+                },
+            },
+            useSlerp: {
+                name: "animationType",
+                inOptions: true,
+                defaultValue: false,
+                dataTransformer: (value) => {
+                    if (value[0] === true) {
+                        return [FlowGraphTypes.Quaternion];
+                    } else {
+                        // TODO - should we return undefined or the default value?
```

Comment:
Does this need to be addressed?

Reply:
Addressed in the block that is handling it.